doc/README

```diff
@@ -13,23 +19,49 @@ optionally, you might need for special features:
  - librsvg2 (for watermark plugin)
  - flickcurl (for Flickr support)
  - libGL/sdl12 (to build slideshow viewer)
- - dbus-glib, gnome-keyring (kwallet/gnome keyring support)
- - osm-gps-map for geo tagging view
+ - dbus-glib
```

Comment:
We no longer use dbus-glib (since commit acf9329921e662efcc158211fe0654b3913ab6ca).